velox/functions/sparksql/SIMDJsonFunctions.h

```diff
@@ -0,0 +1,192 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/functions/prestosql/SIMDJsonFunctions.h"
+
+using namespace simdjson;
+
+namespace facebook::velox::functions::sparksql {
+
+template <typename T>
+struct SIMDGetJsonObjectFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+  std::optional<std::string> formattedJsonPath_;
+
+  // ASCII input always produces ASCII result.
+  static constexpr bool is_default_ascii_behavior = true;
+
+  FOLLY_ALWAYS_INLINE std::string getFormattedJsonPath(
+      const arg_type<Varchar>& jsonPath) {
+    // Makes a conversion from spark's json path, e.g.,
+    // converts "$.a.b" to "/a/b".
```

Comment:
Can we move this comment above `getFormattedJsonPath`, and list all the rules for conversion? E.g. '$' will be ignored, '[' -> '/' etc.